docs/cody/clients/enable-cody-enterprise.mdx

```diff
@@ -67,6 +67,20 @@ Cody Enterprise supports searching up to 10 repositories to find relevant contex
 * In VS Code, open a new Cody chat, type `@`, and select `Remote Repositories` to search other repositories for context
 * In JetBrains, use the enhanced context selector
 
+### @-mention directory
+
+<Callout type="info">@-mentioning directory is available for Enterprise users on VS Code, JetBrains, and Cody Web.</Callout>
+
+To better support teams working with large monorepos, Enterprise users can `@-mention` directories when chatting with Cody. This helps you define more specific directories and sub-directories within that monorepo to give more precise context.
+
+To do this, type `@` in the chat, and then select **Directories** to search other repositories for context in your codebase.
+
+<video width="1920" height="1080" loop playsInline controls style={{ width: '100%', height: 'auto' }}>
+  <source src="https://storage.googleapis.com/sourcegraph-assets/Docs/Media/%40-mention-dir-0824.mp4" type="video/mp4"/>
+</video>
+
+Also, note that you can only `@-mention` remote directories, not the local directory.
```

Comment:
```suggestion
Also, note that you can only `@-mention` remote directories, not local directories.
```

Reply:
Will a user know what a "remote directory" is, or should we say something more like "directories on your Sourcegraph instance"?

Reply:
I think we should say "remote directories", but we can further clarify that this means "directories in your Sourcegraph instance". So I'd say something like: 

> Please note that currently you can only `@-mention` remote directories (directories in your Sourcegraph instance) but not local directories. This means that recent changes to your directories can't be utilized as context until your Sourcegraph instance re-indexes any changes.

> If you want to include recent changes that haven't been indexed in your Sourcegraph instance, you can `@-mention` specific files, lines of code, or symbols.